packages/discovery-provider/ddl/migrations/0044_add_parent_album_ids.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE tracks ADD COLUMN IF NOT EXISTS parent_album_ids integer[];
```

Comment:
Is it that simple?

Reply:
was the query on playlists table too slow? i'm a little hesitant adding this column that does the same thing as playlist_contents. we should probably normalize this relationship in another table.